interface/usergroup/usergroup_admin.php

```diff
@@ -359,9 +359,12 @@
 ?>
 <html>
 <head>
-
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<title><?php xl('User / Group', 'e');?></title>
 <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'];?>/bootstrap-3-3-4/dist/css/bootstrap.css" type="text/css">
+<?php if ($GLOBALS['language_direction'] == 'rtl'): ?>
```

Comment:
i'll bring this fix in; also have a quick style fix for the cancel button in procedure form to bring in, so will bring it with that.